c_tests/test_Reader.cpp

```diff
@@ -541,20 +542,20 @@ TEST_CASE_METHOD(CziBgrCreator2, "test_bgr2_read", "[Reader_read_bgr2]")
   auto pr = imgCont.first->images();
   auto shape = imgCont.second;
 
-  REQUIRE(czi->dimsString() == std::string("SCYX"));
-  std::vector<int> ansSize{ 1, 7, 81, 147 };
+  REQUIRE(czi->dimsString() == std::string("SCYXA"));
+  std::vector<int> ansSize{ 1, 7, 81, 147, 3 };
   REQUIRE(czi->dimSizes() == ansSize);
 
   using DI = pylibczi::DimIndex;
   pylibczi::Reader::DimsShape ansDims{
-    { { DI::S, { 0, 1 } }, { DI::C, { 0, 7 } }, { DI::Y, { 0, 81 } }, { DI::X, { 0, 147 } } }
+    { { DI::S, { 0, 1 } }, { DI::C, { 0, 7 } }, { DI::Y, { 0, 81 } }, { DI::X, { 0, 147 } }, { DI::A, { 0, 2 } } }
```

Comment:
How did this got from 7 channel and no samples to 7 chanel with 2 samples? Or am I misunderstanding this test...

I thought we could only ever have 3 sample images? and I thought you used to expand samples into channels??

Reply:
This is actually an example of why samples are a good thing to include.
The < 3.0 version only returns 21 channels in the numpy array. So this was an inconsistency in the initial version.